src/main/test/data/database/DatabaseManagerTest.java

```diff
@@ -0,0 +1,105 @@
+package data.database;
+
+
+import data.dao.*;
+import data.dao.interfaces.*;
+import data.implementations.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DatabaseManagerTest {
+    private FloorDAO floorDAO;
+    private LocationDAO locationDAO;
+    private NodeDAO nodeDAO;
+    private EdgeDAO edgeDAO;
+    private Location_TagDAO location_tagDAO;
+    private Node node1, node2, node3, node4;
+    private Location location1, location2, location3;
+    private Edge edge1, edge2, edge3;
+    private Floor floor1, floor2;
+    private Location_Tag location_tag1, location_tag2, location_tag3;
+
+
+    @BeforeAll
+    static void init() {
+        DatabaseManager.createNewDatabase();
+    }
+
+    @BeforeEach
+    void setUp() {
+        DatabaseManager.dropTables();
+        DatabaseManager.createTables();
+        floorDAO = new FloorDAOImp();
+        location_tagDAO = new Location_TagDAOImp();
+        locationDAO = new LocationDAOImp();
+        nodeDAO = new NodeDAOImp();
+        edgeDAO = new EdgeDAOImp();
+        floorDAO.insert(floor1 = new Floor(1, "floor1"));
```

Comment:
Myślę, że bardziej przejrzysty kod byłby, gdyby rozdzielić te operacje na:
`floor1 = new Floor(1, "floor1")`
i dopiero wtedy:
`floorDAO.insert(floor1);`
Tyczy się to wszystkich innych miejsc, gdzie tak robicie